### PR TITLE
Analysis tests

### PR DESCRIPTION
**Pull request by mcgallan**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/analysisservices/Azure.ResourceManager.Analysis/tests/AnaltsisManagementTestBase.cs [hunk 1]
```diff
@@ -13,6 +13,8 @@ namespace Azure.ResourceManager.Analysis.Tests
     public class AnaltsisManagementTestBase : ManagementRecordedTestBase<AnaltsisManagementTestEnvironment>
```
Comment (jsquire) [inline sdk/analysisservices/Azure.ResourceManager.Analysis/tests/AnaltsisManagementTestBase.cs:13]:
```suggestion
    public class AnalysisManagementTestBase : ManagementRecordedTestBase<AnaltsisManagementTestEnvironment>
```

Comment (mcgallan) [reply sdk/analysisservices/Azure.ResourceManager.Analysis/tests/AnaltsisManagementTestBase.cs]: fixed

File: sdk/analysisservices/Azure.ResourceManager.Analysis/tests/SessionRecords/AnalysisServiceTests/AnalysticServiceTests().json [hunk 2]
```diff
@@ -0,0 +1,2245 @@
+{
```
Comment (ArthurMa1978) [inline sdk/analysisservices/Azure.ResourceManager.Analysis/tests/SessionRecords/AnalysisServiceTests/AnalysticServiceTests().json:1]: Please move to TestProxy.

File: sdk/analysisservices/Azure.ResourceManager.Analysis/tests/TestCase/AnalysisServiceTests.cs [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+using System;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Analysis.Models;
+using Azure.ResourceManager.Models;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.Analysis.Tests
+{
+    public class AnalysisServiceTests : AnalysisManagementTestBase
```
Comment (ArthurMa1978) [inline sdk/analysisservices/Azure.ResourceManager.Analysis/tests/TestCase/AnalysisServiceTests.cs:13]: Please generate the samples first, then refer the generated samples to add more tests for all resources.

Comment (mcgallan) [reply sdk/analysisservices/Azure.ResourceManager.Analysis/tests/TestCase/AnalysisServiceTests.cs:14]: fixed

File: sdk/analysisservices/Azure.ResourceManager.Analysis/tests/SessionRecords/AnalysisServiceTests/AnalysticServiceTests().json (hunk 2 above)
Comment (mcgallan) [reply sdk/analysisservices/Azure.ResourceManager.Analysis/tests/SessionRecords/AnalysisServiceTests/AnalysticServiceTests().json]: fixed

**Review by ArthurMa1978 — APPROVED** [review]

**Pull request merged**